### PR TITLE
Admin site

### PR DESCRIPTION
**Pull request by twistedtransistor-1947**

### CODE REVIEW

File: apps/pp/admin.py [hunk 1]
```diff
@@ -2,20 +2,61 @@
...
+
+    list_display = ('comment', 'create_date', 'user', 'useful', 'objection', 'priority', 'active')
+    list_editable = ('active',)
+    readonly_fields = (
+        'user', 'url', 'ranges', 'quote', 'create_date', 'priority', 'useful', 'objection', 'reference_request')
+
+    def get_actions(self, request):
+        actions = super().get_actions(request)
+        if request.user.username[0].upper() != 'J':
+            if 'delete_selected' in actions:
```
Comment (kaglowka) [inline apps/pp/admin.py]: Tutaj chyba zostało coś z tutoriala? :)

File: apps/pp/admin.py [hunk 2]
```diff
@@ -2,20 +2,61 @@
...
+class ReferenceRequestAdmin(DjangoQLSearchMixin, SimpleHistoryAdmin):
+    list_display = ('user', 'url', 'active')
+    list_editable = ('active',)
+    readonly_fields = ('user', 'url', 'ranges', 'quote')
+
+    def get_actions(self, request):
+        actions = super().get_actions(request)
+        if request.user.username[0].upper() != 'J':
+            if 'delete_selected' in actions:
+                del actions['delete_selected']
```
Comment (kaglowka) [inline apps/pp/admin.py]: i tutaj :)

**Comment by glowka** [comment]
Shall we close this PR?

**Comment by kaglowka** [comment]
This PR contains an admin feature that we'll very likely need in the future. I'm closing this one, planning to resolve the conflicts and produce a new PR very soon.

**Pull request closed without merging**